pages/linux/logrotate.md

```diff
@@ -0,0 +1,24 @@
+# logrotate
+
+> Rotates, compresses, and mails system logs.
+> More information: <https://manned.org/logrotate.8>.
+
+- Force logrotate to run outside of normal interval:
+
+`logrotate {{path/to/logrotate.conf}} --force`
+
+- Set mail binary for logrotate to Mutt:
+
+`logrotate {{path/to/logrotate.conf}} --mail /usr/bin/mutt`
+
+- Run logrotate without lock or state file:
```

Comment:
```suggestion
- Run without using a state (lock) file:
```

Reply:
I like this a lot better, thank you for the suggestion!